frontend/src/utils.js

```diff
@@ -69,9 +69,35 @@ function gcmCompliant() {
   }
 }
 
+function findMetric(href) {
+  switch(href) {
+    case 'https://www.mozilla.org/':
+      return 'mozilla';
+    case 'https://www.mozilla.org/about/legal':
+      return 'legal';
+    case 'https://testpilot.firefox.com/about':
+      return 'about';
+    case 'https://testpilot.firefox.com/privacy':
+      return 'privacy';
+    case 'https://testpilot.firefox.com/terms':
+      return 'terms';
```

Comment:
Re: #260
Not sure what "privacy" and "terms" means here, or if we need to try clarifying to disambiguate between Send Privacy/Terms and TestPilot Privacy/Terms. Or not, I have no idea what I'm talking about.
/cc @chuckharmston @clouserw